### ISSUE TITLE
fortran free testing

### ISSUE THREAD

**Issue by Shimwell** [pshriwise/double-down#9]
I wonder if it might be worth changing the MOAB compile to without fortran and then removing gfortran from one of the install.sh scripts. and adding some more flags to the MOAB compile

```
                  -DENABLE_FORTRAN=OFF \
                  -DENABLE_BLASLAPACK=OFF \
```

https://github.com/pshriwise/double-down/blob/f297ff989056d12107439129cef7cea53b1a1ef5/ci/install-apt-pkgs.sh#L4

https://github.com/pshriwise/double-down/blob/f297ff989056d12107439129cef7cea53b1a1ef5/ci/moab-install.sh#L16

I can make a PR for this is you think it is worth the effort

**Comment by pshriwise** [comment]
That would be a desirable change and I'd definitely welcome that PR, yes.

**Comment by Shimwell** [comment]
fixed in #10
